test/functorch/test_vmap.py

```diff
@@ -3661,6 +3711,25 @@ def test_vmap_exhaustive(self, device, dtype, op):
         xfail('linalg.lu', ''),
         skip('linalg.ldl_solve', ''),
         skip('_softmax_backward_data'),
+        # One or more of the overload doesn't have a Batch rule.
+        xfail('where'),
+        xfail('bincount'),
+        xfail('bitwise_and'),
+        xfail('bitwise_or'),
+        xfail('bitwise_xor'),
+        xfail('bitwise_left_shift'),
+        xfail('bitwise_right_shift'),
+        xfail('float_power'),
+        xfail('ge'),
+        xfail('gt'),
+        xfail('le'),
+        xfail('lt'),
+        xfail('ne'),
+        # AssertionError
+        # Mismatched elements: 18 / 20 (90.0%)
+        # Greatest absolute difference: 14.031710147857666 at index (0, 5) (up to 0.0001 allowed)
+        # Greatest relative difference: 2.9177700113052603 at index (0, 3) (up to 0.0001 allowed)
+        xfail('narrow_copy', device_type='cpu'),
```

Comment:
Can you file an issue for silent correctness? Also, do you know which of the following is the actual problem?
- the non-contiguous test is failing
- the batching rule is bogus?
- narrow_copy has inconsistent semantics on cpu/cuda?

Reply:
Sure, will file an issue.

* I don't think `non-contiguous` sample is an issue as we haven't added `non-contig` testing to vmap tests.
* Batching rule for `narrow_copy` seems innocuous and doesn't have special handling for CPU and CUDA.

So maybe the operator has some issue.

Batching Rule Ref:
https://github.com/pytorch/pytorch/blob/3120054c151d37fdf43963dbb60ab420908f48cf/aten/src/ATen/functorch/BatchRulesViews.cpp#L506-L515

Reply:
If the operator is a problem: if we can come up with some repro that doesn't involve vmap that shows that on the same input (on cpu/cuda with the same strides), it produces different outputs, then that would be great. One idea to "get rid of the vmap" is to use make_fx to trace out what's happening

Reply:
Sure. Thanks! 

Have assigned the issue to myself. Will have a look soon.